release_bot/releasebot.py

```diff
@@ -279,13 +279,21 @@ def run(self):
                             # There's no way how to tell whether there's already such a fedora 'release'
                             # so try to do it only when we just did PyPi release
                             self.make_new_fedora_release()
+                except ReleaseException as exc:
+                    self.logger.error(exc)
+
+                # Moved out of the previous try-except block, because if it
+                # encounters ReleaseException while checking for PyPi sources
+                # it doesn't check for GitHub issues.
+                try:
                     if self.new_release.get('trigger_on_issue') and self.find_open_release_issues():
                         if self.new_release.get('labels') is not None:
                             self.github.put_labels_on_issue(self.new_pr['issue_number'],
                                                             self.new_release.get('labels'))
                         self.make_release_pull_request()
                 except ReleaseException as exc:
                     self.logger.error(exc)
+                
```

Comment:
this new-line is not needed

Reply:
Gee. I completely forgot about that. :(

Fixed that now.